src/app/report-rollup/phast-report-rollup.service.ts

```diff
@@ -150,9 +150,9 @@ export class PhastReportRollupService {
       } else {
         electricityEnergy += (convertedSumEnergy + convertedEnergySavings);
       }
-      let diffCO2 = result.baselineResults.co2EmissionsOutput - result.modificationResults.co2EmissionsOutput;
+      let diffCO2 = result.baselineResults.co2EmissionsOutput.totalEmissionOutput - result.modificationResults.co2EmissionsOutput.totalEmissionOutput;
```

Comment:
is it ever possible the co2EmissionsOutput can be undefined and accessing the totalEmissionOutput variable throws an error?